chunky/src/java/se/llbit/chunky/renderer/scene/AsynchronousSceneManager.java

```diff
@@ -86,6 +86,13 @@ public void setOnChunksLoaded(Runnable onChunksLoaded) {
       }
     } catch (InterruptedException ignored) {
       // Interrupted.
+    } catch (Throwable e) {
+      if (e instanceof OutOfMemoryError) {
+        Log.error("Chunky has run out of memory! Increase the memory given to Chunky in the launcher.");
+      }
+
+      Log.error("Scene manager has crashed due to uncaught exception:", e);
```

Comment:
```suggestion
      Log.error("Scene manager has crashed due to an uncaught exception. Chunky may not work properly until you restart it. If you think this is a bug, please report it to the developers.", e);
```

Reply:
…or maybe only show the "report bug" thing if the error is _not_ an out of memory error...